### PR TITLE
Added: McNeil site data and apc ip for SCO.

### PR DESCRIPTION
**Pull request by seichold**
This PR covers the following 2 changes:
1. Created a new site mcm to represent the McNeil street HW that is now a core part of the network.
2. Added the IP address and hostname for the APC network card to be installed in SCO for the battery backup.

### CODE REVIEW

File: website/ipam.json [hunk 1]
```diff
@@ -368,10 +368,18 @@
     {
       "address": "44.34.128.222",
       "name": "k4sof",
-      "site": "client",
-      "asset": null,
+      "site": "mcn",
+      "asset": "wlan1-gateway",
```
Comment (turnrye) [inline website/ipam.json:372]: Keep asset null unless you have a corresponding entry on the memhamwan/inventory repo, and in that case you’d put the issue ID here.

```suggestion
      "asset":null,
```

File: website/ipam.json [hunk 2]
```diff
@@ -368,10 +368,18 @@
...
       "os": "routeros",
-      "cidr": null
+      "cidr": 28
+    },
+    {
+      "address": "44.34.129.147",
+      "name": "k4sof",
+      "site": "mcn",
+      "asset": "ether1",
```
Comment (turnrye) [inline website/ipam.json:380]:
```suggestion
      "asset": null,
```

File: website/ipam.json [hunk 3]
```diff
@@ -678,6 +686,38 @@
       "asset": null,
       "os": "ambient-weather",
       "cidr": 28
+    },
+    {
+      "address": "44.34.129.146",
+      "name": "ptpsam.ryan",
+      "site": "mcn",
+      "asset": "ether1-local",
```
Comment (turnrye) [inline website/ipam.json:694]:
```suggestion
      "asset": null,
```

File: website/ipam.json [hunk 4]
```diff
@@ -678,6 +686,38 @@
...
+      "os": "routeros",
+      "cidr": 28
+    },
+    {
+      "address": "44.34.131.146",
+      "name": "ptpsam.ryan",
+      "site": "mcn",
+      "asset": "bridge1",
```
Comment (turnrye) [inline website/ipam.json:702]:
```suggestion
      "asset": null,
```

File: website/ipam.json [hunk 5]
```diff
@@ -678,6 +686,38 @@
...
+      "site": "sco",
+      "asset": null,
+      "os": "apc",
+      "cidr": 28
+    },
+    {
+      "address": "44.34.131.147",
+      "name": "ptpryan.sam",
+      "site": "mcn",
+      "asset": "bridge1",
```
Comment (turnrye) [inline website/ipam.json:718]:
```suggestion
      "asset": null,
```

**Review by turnrye — CHANGES_REQUESTED** [review]
A few small changes just based off of the nuance of how the site is setup.

File: website/ipam.json (hunk 1 above)
Comment (seichold) [reply website/ipam.json:372]: How do we want to account for devices with multiple IPs on different interfaces? It doesn't seem like we have a the ability to address that (and I was cheating using that field).

Comment (turnrye) [reply website/ipam.json:372]: The thing that this is issuing is an “interface” field — asset already tracks device. So for instance in your case, you could add an asset to the repo and then trace your different addresses back to that if you’d like.

This whole list is quick-and-dirty for now. I don’t like this as a long term solution and think that we need to migrate to a real IPAM. But if we can get this data right for now, it’ll make it simple for us to migrate it into an IPAM once we have that in place.

**Comment by turnrye** [comment]
I am closing this PR, as we have stopped using the IPAM page on the site for managing the IPs and instead moved to strictly depending on reverse DNS entries on our DNS server.

**Pull request closed without merging**